frontend/src/components/ColoredInputBars.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { TextInput, Paper, Text, Space } from '@mantine/core';
+import { useViewportSize } from '@mantine/hooks';
+
+function getWidth() {
+    const { width } = useViewportSize();
+    return width;
+  }
+
+function ColoredInputBars(props){
+    const s = getWidth() < 768 ? 'xs' : 'lg';
```

Comment:
Using hardcoded value instead of the mantine style breakpoints, but also could just use repsonsive mantine props:
{{base: 'xs', sm: 'lg'}}